torch/testing/_internal/common_methods_invocations.py

```diff
@@ -287,6 +287,34 @@ def sample_inputs_tensor_split(op_info, device, dtype, requires_grad):
                         args=(torch.tensor([1, 2, 3]),),
                         kwargs=dict(dim=1)),)
 
+def sample_inputs_slogdet(op_info, device, dtype, requires_grad):
+    from torch.testing._internal.common_utils import random_hermitian_matrix, random_hermitian_pd_matrix
+
+    # tests cases are copied from 'method_tests'
+    # but make_nonzero_det is not used, since there is no negative or positive determinant
```

Comment:
Additional cases could be added if the dtype is not complex, though. Would that make sense here?

Reply:
I don't think it is necessary, there is no special handling of the sign value in the code so no need to test specifically the positive or negative signs. In fact, since slogdet_backward is SVD-based we shouldn't have too many test inputs here otherwise it gets expensive very quickly.
I removed a few test cases in b3f8932 which I think are not needed making tests much faster.